spec/controllers/admin/partners_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+RSpec.describe Admin::PartnersController, type: :controller do
+    let(:organization) { create(:organization) }
+    
+    # Use the super_admin instead of organization_admin
+    let(:super_admin) { create(:super_admin) }
+    
+    # Ensure partners are created before the test
+    let!(:partner1) { create(:partner, name: "alpha", organization: organization) }
+    let!(:partner2) { create(:partner, name: "Bravo", organization: organization) }
+    let!(:partner3) { create(:partner, name: "Zeus", organization: organization) }
+  
+    let(:default_params) do
+      { organization_id: organization.id }
+    end
+  
+    context "When logged in as a super admin" do
+      before do
+        sign_in(super_admin)  # Sign in as the super_admin
+      end
+  
+      describe "GET #index" do
+        it "returns http success" do
+          get :index
+          expect(response).to be_successful
+        end
+  
+        it "assigns partners ordered by name (case-insensitive)" do
+          get :index
+          expect(assigns(:partners)).to eq([partner1, partner2, partner3].sort_by { |p| p.name.downcase })
```

Comment:
Suggestion:

```ruby
expect(assigns(:partners).map(&:name)).to eq(%w(alpha Bravo Zeus))
```

But if you really want to test this, you should swap the names of partners 1 and 2. This way the default order would *not* be alphabetical and you'd actually be testing that the sort works. 😄